src/text_shaper/directwrite_shaper.cpp

```diff
@@ -361,25 +361,25 @@ void directwrite_shaper::shape(font_key _font,
             gpos.presentation = _presentation;
             gpos.glyph = glyph_key{ _font, fontInfo.size, glyph_index{glyphIndices.at(i)} };
             // gpos.offset.x = static_cast<int>(cellWidth);
-            //gpos.offset.y = static_cast<int>(static_cast<double>(pos[i].y_offset) / 64.0f);
+            // gpos.offset.y = static_cast<int>(static_cast<double>(pos[i].y_offset) / 64.0f);
```

Comment:
maybe just remove these? if in the future this is needed, we can always add them back.